src/schema/v1/fields/money.ts

```diff
@@ -105,6 +109,37 @@ const money = ({ name, resolve }) => ({
   }),
 })
 
+export const Price = new GraphQLObjectType<any, ResolverContext>({
+  name: "Price",
+  fields: {
+    minor: {
+      type: GraphQLFloat,
+      description: "An amount of money expressed in minor units (like cents).",
+      resolve: ({ cents }) => cents,
+    },
+    currencyCode: {
+      type: GraphQLString,
+      description:
+        "The ISO-4217 alphabetic currency code, as per https://en.wikipedia.org/wiki/ISO_4217",
+      resolve: ({ currency }) => currency,
+    },
+    display: {
+      type: GraphQLString,
+      description: "A pre-formatted price.",
+    },
+    major: {
+      type: GraphQLFloat,
+      description:
+        "An amount of money expressed in major units (like dollars).",
+      resolve: ({ cents, currency }) => {
+        const factor = currencyCodes[currency.toLowerCase()].subunit_to_unit
+        // TODO: Should we round or used a fixed precision?
+        return cents / factor
+      },
+    },
+  },
+})
```

Comment:
Are all these fields meant to be nullable?

Reply:
Nope, good catch.

Reply:
Nullability is updated. 

The only field that's intentionally nullable is the display.The reasoning behind this might not be the best though. 

For the `minPrice` and `maxPrice` in a range, they're not returning a display string from the upstream service. In this case, the upstream service just returns a single display that's `${minPrice} - ${maxPrice}` or something of the sort. Right now, display in the `Price` field for `minPrice` and `maxPrice` is always `null`. Other than that case, we _do_ expect display to be non-nullable so far as I'm aware. 

Is there a better way to handle this?